src/layout/msWorkspace/msWindow.ts

```diff
@@ -505,17 +508,19 @@ export class MsWindow extends Clutter.Actor {
             }
             //Set the size accordingly
             if (isWayland) {
+                const moveTo2 = moveTo;
+                const resizeTo2 = resizeTo;
```

Comment:
Why duplicate the variable there?

Reply:
moveTo has a type that includes undefined, but it is not undefined here. Typescript cannot see that `moveTo` will not be modified to be potentially undefined outside the callback. By using a different variable we make typescript prove that it cannot be undefined inside the callback.